src/loaders.py

```diff
@@ -32,23 +35,26 @@ def __init__(self, ids, annotations, annotations_human_labels, images_dir, targe
         self.target_encoder = target_encoder
         self.train_mode = train_mode
 
+        self.image_transform = image_transform
+
     def __len__(self):
         return len(self.ids)
 
     def __getitem__(self, index):
-
         Xi = self.load_from_disk(index)
 
+        Xi = self.image_transform(Xi)
+
         if self.annotations is not None:
-            yi = self.load_target(index, Xi.size)
+            yi = self.load_target(index, list(Xi.size())[-2:])
```

Comment:
@taraspiotr I would go with

```python
 _,h,w = Xi.size()
yi = self.load_target(index, (h,w))
``` 

to make it more understandable

Reply:
@ agree